examples/tutorial/jupyter/execution/pandas_on_dask/cluster/exercise_5.ipynb

```diff
@@ -0,0 +1,370 @@
+    "directory_path = \"./\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then you need run `dataset_upload` function on all workers. As a result, you will get a dictionary, where the result of the function execution will be for each workers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_path = os.path.join(os.path.abspath(directory_path), \"taxi.csv\")\n",
+    "client.run(dataset_upload, \"https://modin-datasets.intel.com/testing/yellow_tripdata_2015-01.csv\", file_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Yeou need also execute this function on the local machine:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_upload(\"https://modin-datasets.intel.com/testing/yellow_tripdata_2015-01.csv\", file_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<b>Congratulations! The cluster is now fully configured and we can start running Pandas queries.</b>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Executing in a cluster environment\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Same as local mode Modin on cluster uses Ray as an execution engine by default so no additional action is required to start to use it. Alternatively, if you need to use another engine, it should be specified either by setting the Modin config or by setting Modin environment variable before the first operation with Modin as it is shown below. Also, note that the full list of Modin configs and corresponding environment variables can be found in the [Modin Configuration Settings](https://modin.readthedocs.io/en/stable/flow/modin/config.html#modin-configs-list) section of the Modin documentation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Modin engine can be specified either by config\n",
+    "import modin.config as cfg\n",
+    "cfg.Engine.put(\"dask\")\n",
+    "\n",
+    "# or by setting the environment variable\n",
+    "# import os\n",
+    "# os.environ[\"MODIN_ENGINE\"] = \"dask\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now you can use Modin on the Dask cluster.\n",
+    "\n",
+    "Let's read the downloaded CSV file and executes such pandas operations as count, groupby and map:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import modin.pandas as pd\n",
+    "\n",
+    "df = pd.read_csv(file_path, quoting=3)\n",
+    "df_count = df.count()\n",
+    "df_groupby_count = df.groupby(\"passenger_count\").count()\n",
+    "df_map = df.map(str)\n"
```

Comment:
Let's also put lines with measuring this.

Reply:
Done